deploy/src/main/java/org/nem/deploy/NemFormatter.java

```diff
@@ -14,7 +15,8 @@ public class NemFormatter extends SimpleFormatter {
 
 	@Override
 	public synchronized String format(final LogRecord record) {
-		record.setMillis(TIME_PROVIDER.getNetworkTime().getRaw() + SystemTimeProvider.getEpochTimeMillis() - TIME_ZONE_OFFSET);
```

Comment:
since `setMillis` method was deprecated, replaced it with `setInstant` as suggested.